src/Router/AppRouter.tsx

```diff
@@ -364,6 +358,7 @@ const routes = {
   "/facility/:facilityId/patient/:patientId/consultation/:consultationId/treatment-summary":
     ({ patientId, consultationId }: any) => (
```

Comment:
```suggestion
    ({ facilityId, patientId, consultationId }: any) => (
```

_do destructure facilityId from the param before using_